db/migrations/6.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE results;
+CREATE TABLE results (
+  id SERIAL,
+  driver_id INTEGER,
+  race_id INTEGER,
+  team_id INTEGER,
+  year TEXT,
+  round TEXT,
```

Comment:
I've set `year` and `round` to text types because I don't think there's ever a situation where we need to do maths on these values - what do you think?